zokrates_core/src/typed_absy/mod.rs

```diff
@@ -513,6 +513,10 @@ pub enum BooleanExpression<'ast, T: Field> {
         Box<FieldElementExpression<'ast, T>>,
         Box<FieldElementExpression<'ast, T>>,
     ),
+    EqBool(
```

Comment:
I think for consistency, if we call this `EqBool` we should rename `Eq` to `EqField`.

I also have a preference for `BoolEq` like "boolean equality".